lib/Parser/Ntriples.php

```diff
@@ -7,6 +7,7 @@
  * LICENSE
  *
  * Copyright (c) 2009-2013 Nicholas J Humfrey.  All rights reserved.
+ * Copyright (c) 2020 Austrian Centre for Digital Humanities.
```

Comment:
Is accepting this Pull Request conditional on assigning copyright ownership to the Austrian Centre for Digital Humanities?

I am not sure about the legality of the copyright of Open Source contributions, but it would seem to make things easier to have a single copyright owner.

I have no intension of changing the licensing terms aways from MIT. But having multiple copyright owners would seem to me to make things more complex.

Having "Copyright The EasyRdf Team" might be an option, but given that it isn't a legal entity, I am not sure if it is meaningless.

Reply:
I will ask people who are paiyng for my work but I guess they won't push on it.

By the way the README and the composer.json state the license is BSD-3. (although in practice it's not much difference from MIT)

Reply:
We can get rid of "Austrian Centre for Digital Humanities" copyright headers.